packages/fxa-graphql-api/src/config.ts

```diff
@@ -44,7 +44,19 @@ const conf = convict({
   contentfulApiKey: {
     doc: 'GraphQL Content API key for Contentful hCMS to fetch RP-provided content',
     format: String,
-    env: 'CONTENTFUL_API_KEY',
+    env: 'CONTENTFUL_GRAPHQL_API_KEY',
+    default: '',
+  },
+  contentfulSpaceId: {
+    doc: 'Alphanumeric id used for instantiating the ContentfulClient',
+    format: String,
+    env: 'CONTENTFUL_GRAPHQL_SPACE_ID',
+    default: '',
+  },
+  contentfulEnvironment: {
+    doc: 'Environment alias used for instantiating the ContentfulClient',
```

Comment:
From the [environment alias docs](https://www.contentful.com/developers/docs/concepts/environment-aliases/), it sounds like this could be the environment ID OR the environment alias. Might be worth linking to Contentful's [environment](https://www.contentful.com/developers/docs/concepts/multiple-environments/) docs?